### ISSUE TITLE
Forum upduck button wraps poorly

### ISSUE THREAD

**Issue by williamjallen** [Submitty/Submitty#11175]
**Describe the bug**
The forum upduck currently looks like this:
![image](https://github.com/user-attachments/assets/a36bd93b-803c-4063-8772-ac61becd0197)

**Expected behavior**
The duck icon should always be on the same line as the upduck count and the reply button, regardless of the screen size.

**Comment by ShashidharM0118** [comment]
I noticed that "upduck" appears on the same line across all screen sizes. Has this issue already been addressed?

**Comment by himanshutalekar03** [comment]
Has this issue been resolved? If not, I would be happy to work on it.

**Comment by lalithaar** [comment]
Hey this is to document, I am taking up this issue

**Comment by jeffrey-cordero** [comment]
I don't believe this is still an issue on the forum. Perhaps improved row-spacing for the buttons on small screens would be helpful.

![Image](https://github.com/user-attachments/assets/95d9acdc-b0dd-46be-86a3-941cc9b6c642)

**Comment by bmcutler** [comment]
probably fixed :)
